canonical-websites/docs.ubuntu.com
# Update to latest run script and talisker

Fixes the [broken builds](https://jenkins.canonical.com/webteam/job/deploy-to-docs.staging.ubuntu.com/95/console),

Also, driveby updates documentation-builder to latest, to use latest template for docs.

QA
--

``` bash
./run clean
./run
```

Browse around the site, check things look fine. Then check the docker image:

``` bash
docker build --build-arg COMMIT_ID=`git rev-parse HEAD` --tag docs .
docker run -ti -p 8099:80 docs
```

Now go to http://localhost:8099 and check the site works.

Go to http://0.0.0.0:8099/snap-enterprise-proxy/, http://0.0.0.0:8099/landscape/ etc., and check the documentation pages look okay.
